src/control/KinematicSimulationTrajectoryExecutor.cpp

```diff
@@ -105,8 +105,9 @@ void KinematicSimulationTrajectoryExecutor::step()
     mInExecution = false;
   }
 
+  using seconds = std::chrono::duration<float>;
```

Comment:
Nit: I believe the type of time should be `double` as `Trajectory` takes it as `double`.